### PR TITLE
Add Release Version + Removed Emojis

### PR DESCRIPTION
**Pull request by kquirapas**
Changes
1. [Removed Emojis](https://github.com/web3phl/directory/pull/60/commits/18e88b165c4dbcb8567375fa38697e7c41eeadb0) for cleaner and more professional UI. (Emojis can stay in READMEs and Commit logs)
2. [Added Release Number in Footer](https://github.com/web3phl/directory/pull/60/commits/055fc59c85895050ba85d755ea046cfd1fb56385) to address #37

### CODE REVIEW

**Comment by kquirapas** [comment]
Let me know if I need to change the way I added it on footer.

**Comment by warengonzaga** [comment]
I think you can leave the 🖥️💖☕ by ... in the footer it represents the "Code with Love with Coffee" then at the new line add the "Directory v1.2.0" also add links to the "Web3 Philippines" and "Wareneutron Developers" (use the official link `wareneutron.com`)

For the version text use this link `https://github.com/web3phl/directory/releases/latest`

**Review by warengonzaga — CHANGES_REQUESTED** [review]
Just keep the emoji in the footer and few notes.

**Comment by kquirapas** [comment]
Copy @warengonzaga

File: src/pages/index.tsx [hunk 1]
```diff
@@ -59,7 +59,8 @@ const Home: NextPage = () => {
       <footer>
         <div className="flex justify-center">
           <p className="mb-[20px] text-center text-base leading-6 text-neutral-light md:mb-[70px] md:text-left">
-            Web3 Philippines &amp; Wareneutron Developers
+            <strong>Directory v1.2.0</strong> by Web3 Philippines &amp;
```
Comment (jnebab) [inline src/pages/index.tsx:62]: we probably don't need to make the version number to be bold here. a shade same with the last updated text should be fine here.

**Review by jnebab — CHANGES_REQUESTED** [review]

File: src/pages/index.tsx (hunk 1 above)
Comment (kquirapas) [reply src/pages/index.tsx:62]: got it thanks for the input Ivan!

**Comment by kquirapas** [comment]
@jnebab @warengonzaga Requested changes pushed

**Comment by kquirapas** [comment]
Here's a screenshot for easy preview:

image

**Review by jnebab — APPROVED** [review]
lgtm

**Review by warengonzaga — APPROVED** [review]
LGTM 💪

**Pull request merged**